README.md

```diff
@@ -562,3 +562,19 @@ $config = [
 $ydb = new \YdbPlatform\Ydb\Ydb($config);
 ```
 
+## gRPC options
+
+You can customize the gRPC client's behavior by setting options in config array
+
+Example of using:
+```php
+$config = [
+    // ...
+    'opts' => [
```

Comment:
it will be better to separate grpc options to own section: 'grpc' instead of common 'opts'

Reply:
fixed